week4/fuel.py

```diff
@@ -0,0 +1,16 @@
+def main():
+    fuel = ("Fraction: ")
+    try:
+        num, div = fuel.split("/", 1)
```

Comment:
With Python, do you need to cast `num` and `div` to numbers like integer?  the `.split()` function will return strings, right?

Reply:
done